src/App.tsx

```diff
@@ -9,56 +9,102 @@ import { PostDetails } from './components/PostDetails';
 import { UserSelector } from './components/UserSelector';
 import { Loader } from './components/Loader';
 
+// Context
+import { Provider } from './components/Context';
+
+// Types
```

Comment:
```suggestion
import { Provider } from './components/Context';

```
Do not leave comments